frontend/src/app/features/work-packages/components/wp-edit/work-package-changeset.ts

```diff
@@ -11,6 +11,13 @@ export class WorkPackageChangeset extends ResourceChangeset<WorkPackageResource>
     if (key === 'project' || key === 'type') {
       this.updateForm();
     }
+
+    // Emit event to notify Stimulus controller in activities tab in order to update the activities list
+    // TODO: emit event when change is persisted
+    // currently the event might be fired too early as it only reflects the client side change
+    document.dispatchEvent(
+      new CustomEvent('work-package-updated'),
```

Comment:
Instead of throwing around events, you could also get the controller and call actions on it: https://stimulus.hotwired.dev/reference/controllers#directly-invoking-other-controllers

Reply:
I wasn't aware this is possible for communicating from angular to stimulus. Additionally, I don't see a big issue in using the event approach in this case. Do you much prefer not to use an event here?